timeless-api/src/main/webui/src/app/guards/auth_guard.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import {
+  CanActivate,
+  CanActivateChild,
+  Router,
+  ActivatedRouteSnapshot,
+  RouterStateSnapshot,
+  UrlTree,
+} from '@angular/router';
+import { timelessLocalStorageKey } from '../constants';
+
+@Injectable({ providedIn: 'root' })
+export class AuthGuard implements CanActivate, CanActivateChild {
+  constructor(private readonly router: Router) {}
+
+  private isLogged(): boolean {
+    return localStorage.getItem(timelessLocalStorageKey) != null;
```

Comment:
Hi @matheusandre1, we have a problem here, we can easily set a key with the same name in the localStorage and this code will return true.

Let's verify if the token is valid in the backend, WDYT?